.github/workflows/guix-build.yml

```diff
@@ -59,11 +59,22 @@ jobs:
             echo "Guix build failed!"
             exit 1
           fi
-      - name: Compute SHA1 checksum
+      - name: Compute SHA1 checksums for binaries
```

Comment:
Any reason why we're computing SHA1 hashes at all, considering SHA1 is known to be broken, not just theoretically but [in practice as well](https://shattered.io/static/shattered.pdf) with Git 2.19 onwards [implementing](https://github.com/git/git/commit/34f229790dcc7c7dc61bc37532f88fca39e36de2) SHA256 support.

While still expensive to pull off in the real world, I do think SHA1 hashing should be dropped altogether.

Reply:
Probably; for some reason when we initially were checking guix hashes against each others, we were using sha1 for some reason. I'd be fine moving to sha256 all together.

Reply:
> for some reason when we initially were checking guix hashes

```
$  sha1sum ....
6e8fd5055868b90fb94c221c73fef74bd8181f63  distsrc-x86_64-w64-mingw32/src/dashd.exe
$ sha256sum ...
87428fc522803d31065e7bce3cf03fe475096631e5e07bbd7a0fde60c4cf25c7 distsrc-x86_64-w64-mingw32/src/dashd.exe
```

That's me used here first time sha1 instead of sha256: https://github.com/dashpay/dash/pull/5194
There is no good reason for it, sha1sum is just first in my auto-complete list and its output looks more tidy-up for debug.

Let's move to `sha256sum`!

Reply:
👍 
done